src/components/connections.vue

```diff
@@ -9,14 +9,18 @@ export default {
   data() {
     return {
       name: connectionName,
-      connectionString
+      connectionString,
+      showConnections: true
```

Comment:
The name `showConnections` suggests that it's toggling the entire list of connections, but in this case it should toggle only the add new connection form. It might make sense to change the name to something like `showAddNewConnectionForm` or similar.